blockchain/types/anchoring_data.go

```diff
@@ -67,7 +66,7 @@ type AnchoringDataInternalType0 struct {
 	ReceiptHash   common.Hash
 	StateRootHash common.Hash
 	BlockNumber   *big.Int
-	Period        *big.Int
+	BlockCount    *big.Int
```

Comment:
What does it for?

Reply:
The field was changed to indicate how many blocks the cumulative tx count was calculated.